### ISSUE TITLE
limiting cores while training , training on 3 channel images 

### ISSUE THREAD

**Issue by anorak94** [mpicbg-csbd/stardist#54]
1) is there an option to limit core usage during training or do i have to do it by setting keras and 
    tensorflow environment variables 
2) Is it possible to train on 3 channel images and 1 channel mask, i notice that an assert statement 
     checks for that 
3) how do you generate 3 channel images of the mask, is it okay to do it by concatenating 1 
    channel mask images 3 times or does labkit have an option to do that

**Comment by maweigert** [comment]
> 1. is there an option to limit core usage during training or do i have to do it by setting keras and tensorflow environment variables

You can limit the number of cores by setting the environment variable `OMP_NUM_THREADS` before importing everything else, like so

```python
import os
os.environ["OMP_NUM_THREADS"] = "2"
```

> 2. Is it possible to train on 3 channel images and 1 channel mask, i notice that an assert statement checks for that

Yes, currently only single channel output (i.e. a single class of objects) is supported.

> 3. how do you generate 3 channel images of the mask, is it okay to do it by concatenating 1
>     channel mask images 3 times or does labkit have an option to do that

As before, having multi-channel masks is currently not supported.
